flybywiresim/a32nx
# feat: Colored INOP labels based on github branch

## Summary of Changes

### Colored Label

Adds copying of DDS files into the aircraft folder based on the current branch to the build.js.

This added two DDS files to the `src/Textures/decals 4k` folder 
- one Stable and Development: A320NEO_COCKPIT_DECALSTEXT_ALBD.TIF-yellow.dds
- one for all other branches: A320NEO_COCKPIT_DECALSTEXT_ALBD.TIF-red.dds

This will help recognize the version user uses by simply looking at the screenshot. This will be very helpful for the support team. 

The effect in the cockpit is: 
- No Sticker = Stable
- Yellow = Development
- Red = Experimental and other dev branches

Side note: Once WX and radar are operational again we should move the sticker and add another text. 

### Livery folder rename

This PR also renames the default liveries which has the effect that they are now sorted first on most users' systems and therefore also become default after switch aircraft.
It also fixes the meta data in the Aircraft Selection in development mode showing FlyByWire as the creator instead any other first livery's meta data. 

The change only adds a single "_" to the folder name so it if users have a livery also with a folder name starting with "_" it might be sorted before our default livery. 

## Screenshots (if necessary)
INOP Label:
![image](https://user-images.githubusercontent.com/16833201/147880536-3268b06d-709c-46d2-91c5-84ab86b0eae0.png)
![image](https://user-images.githubusercontent.com/16833201/147880549-5adae113-e692-4063-ba7c-8f234c3fb98b.png)

Livery before:
![image](https://user-images.githubusercontent.com/16833201/147927397-0fb537ef-e6c3-4b61-b71a-031b59093d40.png)

Livery after:
![image](https://user-images.githubusercontent.com/16833201/147927423-814d98dc-0f4c-467d-b527-d17b95fa0804.png)
![image](https://user-images.githubusercontent.com/16833201/147927432-1ebac5ea-9ac7-40d7-89d3-77af99efe668.png)
![image](https://user-images.githubusercontent.com/16833201/147927915-58d046b1-5809-444f-a94b-d3b88e087913.png)

## References
N/A

## Additional context
One of the most common questions in the Discord #support channel is which version is used and to provide a screenshot. 
With this change it is immediately apparent which version the users uses and if it is supported.


Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions

- Make sure the INOP label is red with white font. 
  - Stable and Development can't be tested during PR QA - needs a developer to test the code. 

- Check if default livery is sorted first in the livery list. If not check the folder name of the livery sorted first and confirm if it is alphabetically sorted before "_FlyByWire_A320_NEO-LIVERY"

Check if you can find any issues with selecting liveries and also start a flight if the default liveries are working. 
Should not have any impact on flight but can't hurt do a flight with this PR.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
